src/install-otp

```diff
@@ -2,11 +2,18 @@
 
 set -eo pipefail
 
-cd $RUNNER_TEMP
-
-wget -q -O otp.tar.gz https://repo.hex.pm/builds/otp/${2}/${1}.tar.gz
-mkdir -p .setup-elixir/otp
-tar zxf otp.tar.gz -C .setup-elixir/otp --strip-components=1
-rm otp.tar.gz
-.setup-elixir/otp/Install -minimal $(pwd)/.setup-elixir/otp
-echo "$(pwd)/.setup-elixir/otp/bin" >> $GITHUB_PATH
+cd "$RUNNER_TEMP"
+
+OS=${1}
+FILE_INPUT=${2}.tar.gz
```

Comment:
Love the clarity here, I was just looking at this 💯

Reply:
Yeah, I feel $1, $2, etc. in `bash` doesn't really help, most of the time.